cockatrice/src/dlg_connect.cpp

```diff
@@ -162,8 +162,7 @@ DlgConnect::DlgConnect(QWidget *parent) : QDialog(parent)
 
     previousHostButton->setChecked(true);
 
-    connect(previousHosts, SIGNAL(currentIndexChanged(const QString &)), this,
-            SLOT(updateDisplayInfo(const QString &)));
+    connect(previousHosts, SIGNAL(currentTextChanged(const QString &)), this, SLOT(updateDisplayInfo(const QString &)));
```

Comment:
why change this? the signal didn't disappear

Reply:
also.. conflicts with my pr...

Reply:
ah, I see, it takes an int now, I already changed this in my pr

Reply:
I want this pr merged first

Reply:
Gotcha, so I'll resolve this comment for now